app/helpers/application_helper.rb

```diff
@@ -223,7 +223,7 @@ def timeframe_check(given_timeframe)
   end
 
   def list_path
-    return "" if params[:tag].blank?
+    return "" if params[:tag].blank? || params[:feed_type].blank?
```

Comment:
This is a temporary hack.

Reply:
Suggestion: Comment here along the lines of `Remove when ... `